src/common/utils.js

```diff
@@ -546,3 +547,18 @@ export const getCoresPerSocketPossibilities = (maxCoresPerSocket, vCPUs) => {
 
   return options
 }
+
+export const getReadableRandomString = (() => {
```

Comment:
Add a comment explaining what this function does and for what it should be used.